### DISCUSSION TITLE
FIFO by Server?

### DISCUSSION THREAD

**Discussion by mattbaylor** [postalserver/postal#2277] [Development discussions]
Hi All,

we've started using postal to deliver some fairly sizable outbound emails from multiple clients, however we're running into a bit of a snag: when we get a large send from one client ~10000 outbound emails, the server chugs on it for a while and all other incoming requests sit in line behind the rather large outbound queue. Is there anyway to adjust the queuing to be FIFO by Postal Org/Server instead of FIFO by Postal Installation?

**Comment by willpower232** [comment]
I don't believe this is currently possible, I imagine the implementation would involve multiple distinct queues from RabbitMQ and then allocating workers to each one so not a quick setup but I don't know enough about RabbitMQ so there might be a more straightforward option
